data/types/bolus/combination/combination.go

```diff
@@ -96,7 +101,7 @@ func (c *Combination) Validate(validator structure.Validator) {
 			// If Normal is zero, then _either_:
 			if c.Extended != nil {
 				if c.NormalExpected == nil {
-					validator.Float64("extended", c.Extended).GreaterThan(ExtendedMinimum)
+					validator.Float64("extended", c.Extended).GreaterThanOrEqualTo(ExtendedMinimum)
```

Comment:
In the case where
- `normal` is zero,
- `NormalExpected` doesn't exist, and
- `extended` is zero
`ExtendedExpected should be greater than `ExtendedMinimum`

Same for when
- `normal` is zero,
- `extended` is zero
- `extendedExpected` doesn't exist
`NormalExpected should be greater than `ExtendedMinimum`

Or are we checking this somewhere else and I'm missing it?

Reply:
Neither of those cases are covered by the current validation. If first three conditions are zero we check that `ExtendedExpected` is in range [0,100] for the first example and `NormalExpected` is in range [0,100] for the second example.

Reply:
I added failing tests for those 2 cases and will work on a fix later today.